torch/nn/functional.py

```diff
@@ -2299,6 +2299,24 @@ def pairwise_distance(x1, x2, p=2, eps=1e-6, keepdim=False):
     return torch.pairwise_distance(x1, x2, p, eps, keepdim)
 
 
+pdist = _add_docstr(torch.pdist, r"""
+pdist(input, p=2) -> Tensor
+
+Computes the p-norm distance between every pair of row vectors in the input.
```

Comment:
Since this differs in the parameters from scipy (doesn't have a 'metric'), this should say something like "This is equivalent to scipy.spatial.distance.pdist with metric='minkowski'".

Reply:
I considered this, but I wondered how much of a reference you wanted to have to scipy in the docs. It might become invalid as scipy changes, and as you point out below, it doesn't always map to the "right" value. I'm going to assume that you still want me to go ahead with this change unless noted otherwise.

Reply:
Good question!  I think in this case it is justified because the function has the exact same name and we are talking about parameter names, which would have to change in a backwards-incompatible way for our docs to be invalid.